cairo/src/runner.rs

```diff
@@ -0,0 +1,472 @@
+//! This module represents a run of a Cairo program as a consecution of execution
```

Comment:
-> as a "series" of execution steps?

Reply:
👍  or series of consecutive execution steps, I think that's what I wanted to say and my English failed